index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>George M'sapenda Portifolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+</head>
+<body>   
+    <main class="container">
+        <header class="header">
+            <nav class="navigation">                
+                <p class="logo">George M'sapenda</p>
+                <button class="btn">
+                    <img src="/images/Icon - Menu.png" alt="menu bar icon">
+                </button>                    
+                </div>
+            </nav>
+        </header>
+        <section class="intro">
+            <h1 class="intro_greeting">I'm George<br> Nice to see you!</h1>
+            <p class="intro_text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <p class="connect">LET'S CONNECT</p>
+            <div class="socials">
+                <a href="#">
+                    <img src="images/Github icon.png" alt="Github link">
+                </a>
+                <a href="#">
+                    <img src="images/Linkedin icon.png" alt="Linkedn link">
+                </a>    
+                <a href="#">
+                    <img src="images/Twitter icon.png" alt="Twitter link">
+                </a>
+            </div>
```

Comment:
[ Required ] Please wrap the social media icons in a `ul` tag and each social media icon in a `li` tag that holds a link to the social media page as shown below. A list of icons is more friendly to screen readers and makes webpages to be accessible to persons living with disabilities.

Please remember to include links to your social media profiles in the `href` attribute of the `a` tag, this will increase chances of recruiters engaging you on social media platforms.

## Example
```HTML
<ul class="socials">
        <li> 
             <a href="#">
                <img src="images/Github icon.png" alt="Github link">
             </a>
         </li>
         <li> 
              <a href="#">
                    <img src="images/Linkedin icon.png" alt="Linkedn link">
                </a>
        </li>
        <li> 
             <a href="#">
                <img src="images/Twitter icon.png" alt="Twitter link">
             </a>
         </li>
    </ul>
```